ui/sign-in.reel/sign-in.js

```diff
@@ -85,7 +89,7 @@ var SignIn = exports.SignIn = AuthorizationPanel.specialize({
         value: function () {
             this.addEventListener("action", this, false);
             this._keyComposer.addEventListener("keyPress", this, false);
-
+            this.isBrowserSupported = !!window.chrome && !!window.chrome.webstore;
```

Comment:
There is an environment object in Montage that can help here.
https://github.com/montagejs/montage/blob/master/core/environment.js